flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/internal/KafkaConsumerThread.java

```diff
@@ -245,6 +238,23 @@ public void run() {
 				if (records == null) {
 					try {
 						records = consumer.poll(pollTimeout);
+						// register Kafka's very own metrics in Flink's metric reporters
+						if (useMetrics && !records.isEmpty()) {
+							// register Kafka metrics to Flink
+							Map<MetricName, ? extends Metric> metrics = consumer.metrics();
+							if (metrics == null) {
+								// MapR's Kafka implementation returns null here.
+								log.info("Consumer implementation does not support metrics");
```

Comment:
The log will be overloaded with these, if the MapR implementation is used and metrics is turned on.

Reply:
i change the level to debug.